tests/utils/test_utils.py

```diff
@@ -618,7 +618,7 @@ def test_image_name_comparison():
     (11, None)
 ))
 def test_user_warnings_handler(message, expected):
-    def mocked_log(level, message):
+    def mocked_log(level, message, args):
```

Comment:
I consider this test wrong because it already misses issue instead of catching it, I think test should use `caplog` fixture instead of mocking

Reply:
+1

Reply:
The test has been changed